WordPress/Classes/ViewRelated/Reader/Manage/ReaderManageScenePresenter.swift

```diff
@@ -1,5 +1,41 @@
 class ReaderManageScenePresenter: ScenePresenter {
 
+    enum TabbedSection {
+        case tags
+        case sites
+
+        private func makeViewController() -> UIViewController {
+            switch self {
+            case .tags:
+                return ReaderTagsTableViewController(style: .grouped)
+            case .sites:
+                return ReaderFollowedSitesViewController.controller(showsAccessoryFollowButtons: true, showsSectionTitle: false)
+            }
+        }
+
+        var tabbedItem: TabbedViewController.TabbedItem {
+            switch self {
+            case .tags:
+                return TabbedViewController.TabbedItem(title: NSLocalizedString("Followed Tags", comment: "Followed Tags Title"),
+                                                               viewController: makeViewController(),
+                                                               accessibilityIdentifier: "FollowedTags")
+            case .sites:
+                return TabbedViewController.TabbedItem(title: NSLocalizedString("Followed Sites", comment: "Followed Sites Title"),
+                                                                viewController: makeViewController(),
+                                                                accessibilityIdentifier: "FollowedSites")
```

Comment:
kinda nitpick, but can we pass the `TabbedSection` value to the `TabbedItem` initializer and let that `struct` figure it out the rest, since the real difference are the words "Sites" and "Tags"? Maybe we could assign a `String` `rawValue` to the `enum` to facilitate this?

Reply:
I left this as is for now because I wanted the `TabbedItem` type to be unaware of `TabbedSection` (which is more specific to this filter implementation). I could rearrange some things to make this more of a formula or add an extension to `TabbedItem` but I'm not sure it would help all that much.